eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/persistence/DbsnpSubmittedVariantAccessioningDatabaseService.java

```diff
@@ -137,4 +137,8 @@ private DbsnpSubmittedVariantEntity getNewestVersion(List<DbsnpSubmittedVariantE
         }
         return lastVersionEntity;
     }
+
+    public AccessionWrapper<ISubmittedVariant, String, Long> getLastInactive(Long accession) {
+        return SubmittedVariantAccessioningDatabaseService.getLastInactive(accession, inactiveService);
```

Comment:
it doesn't belong to the current parent (BasicSpringDataRepositoryMonotonicDatabaseService) but I can certainly extract all the common parts of DbsnpSubmittedVariantAccessioningDatabaseService and SubmittedVariantAccessioningDatabaseService which is, the second class is the same as half of the first class.

Otherwise I can just copy the code twice. I was concerned that the code evolves in different directions, but given the difference in cardinality (of documents given a single accession) I guess it's ok for them to evolve differently

Reply:
I duplicated the code